### ISSUE TITLE
[IDEA] ERROR: could not find driver - Should be more explained

### ISSUE THREAD

**Issue by yassinrais** [phalcon/phalcon-devtools#1479]
When a Database Driver Extension is missed

its should say 
**ERROR: could not find php extension driver _postgress_**

because its take more time to understand that the extension who is missed and not an issue about the name of adapater

Thanks :D !

**Comment by BeMySlaveDarlin** [comment]
This is base message related to PHP core \Exception
```
} catch (Exception $e) {
    fwrite(STDERR, 'ERROR: ' . $e->getMessage() . PHP_EOL);
    exit(1);
}
```
As its created outside of phalcon, we cannot properly customize message.
Available options is: 
> We can change message to `DATABASE ERROR: could not find driver`, catching directly PDOException
> We can add trace of exception after message to output

**Comment by Jeckerson** [comment]
@BeMySlaveDarlin Ok, we will return to this in next major version.

**Comment by Ultimater** [comment]
Devtools provides tools which can save a developer a few minutes.
But a vague error like this will send a developer on a totally avoidable quest trying to figure out what's wrong.
In the spirit of devtools, where we want to save people time, I believe this issue is very important to fix.

I'll work on releasing a fix for this on 3.4.x, 4.0.x, and 4.1.x

**Comment by Ultimater** [comment]
This is fixed in v3.4.14, v4.0.8 and v4.1.1
